packages/wrangler/src/triggers/index.ts

```diff
@@ -57,7 +57,7 @@ async function triggersDeployHandler(
 ) {
 	await printWranglerBanner();
 
-	const config = readConfig(undefined, args);
+	const config = readConfig(args);
```

Comment:
This is a change in behaviour, but it looks like the previous behaviour must've been a bug

Reply:
Oh yeah, you're right. Didn't spot that.